CHANGES/665.doc

```diff
@@ -0,0 +1,3 @@
+Fixed broken internal references to :meth:`~URL.human_repr()`.
+Fixed broken external references to :doc:`multidict:index` docs.
+Minor styling improvements.
```

Comment:
Looks like there are 3 separate notes so they should be in separate files and contain information meaningful to the end-users facing the changelog for the first time. They should tell concrete things, describe the effects of updating from the previous version.
Because the changelog describes the things that happened since the previous release, in general, it should use something like the past tense.
The changelog shouldn't refer to internal implementation details that the end-users aren't expected to be familiar with. Telling them that there were arbitrary unspecified improvements of an arbitrary concept of style (writing style? code style? visual/UX/docs theme style?) is not particularly helpful.

Looks like the first two lines are reasonably useful, I'd put them into two separate files. To avoid filename collision, Towncrier supports files with numeric suffixes after the type, like `665.doc.1` + `665.doc.2` (or `665.doc.1.rst` + `665.doc.2.rst`).